.github/workflows/rosa-cluster-create.yml

```diff
@@ -82,7 +82,7 @@ jobs:
         if: always()
         run: |
           mkdir -p provision/aws/logs
-          rosa logs install --debug -c ${{ inputs.clusterName }} > provision/aws/logs/rosa-logs-${{ inputs.clusterName }}
+          rosa logs install --debug -c ${{ inputs.clusterName }} &> provision/aws/logs/rosa-logs-${{ inputs.clusterName }}
```

Comment:
@ryanemerson agree, I have tried it and it works as expected. The only question is, in case of `rosa_delete_cluster.sh` the logs will be combined  in output (hcp and rosa logs). If that's fine, then I will make that change.

Reply:
I think the combination is better than no logs in the worst-case scenario. In the best case, the logs will upload as expected and we can just view the file directly.

Reply:
@ryanemerson thanks, so I have updated the PR as you have suggested.